docs/changelog/CHANGELOG.md

```diff
@@ -42,6 +42,8 @@ See also [LTS releases](https://docs.victoriametrics.com/lts-releases/).
 * BUGFIX: [vmsingle](https://docs.victoriametrics.com/single-server-victoriametrics/) and [vmagent](https://docs.victoriametrics.com/vmagent/): properly escape IPv6 address in [Kubernetes service-discovery](https://docs.victoriametrics.com/sd_configs/#kubernetes_sd_configs) with `role: pod` for containers without exposed ports. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8374).
 * BUGFIX: [vmalert-tool](https://docs.victoriametrics.com/vmalert-tool/): clean up the temporary storage path when process is terminated by SIGTERM or SIGINT. Previously, unclean shut down might affect the next run.
 * BUGFIX: [vmui](https://docs.victoriametrics.com/#vmui): fix an infinite loader on the [Downsampling filters debug page](https://docs.victoriametrics.com/#vmui) when provided configuration matches no series. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8339).
+* BUGFIX: [vmsingle](https://docs.victoriametrics.com/single-server-victoriametrics/), `vmselect` in [VictoriaMetrics cluster](https://docs.victoriametrics.com/cluster-victoriametrics/): prevent panic when using `@` modifier with series which is not aligned to the provided time range. Previously, this was causing a panic in case of a missing value. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/8444).
```

Comment:
the description doesn't seem clear to me as a reader. In other words, it doesn't help me to understand if i am affected without checking the ticket.

Reply:
I've tried to make it more clear, but can't find any phrase which could explain this better without providing a full example. Any ideas on better message here?

Reply:
This is one of the reasons why I asked for test.
The description of change says about misalinged series. While code says it is about getting NaN instead of value or timestamp. So I wanted to see tests to avoid confusion.